Automattic/wp-calypso
# bulk domains page - update domains table checkbox and font size

Related to # Automattic/dotcom-forge#7073

## Proposed Changes

* Updates the font size of the domain name to .875 REM (14px)
* Updates the checkbox size to 16px
* This is all applied underneath the parent selector of `.is-bulk-domains-page`

BEFORE
<img width="373" alt="Screenshot 2024-05-10 at 9 18 33 AM" src="https://github.com/Automattic/wp-calypso/assets/28742426/226811f7-447c-47c4-8ea8-a91329aac9cb">


AFTER

<img width="264" alt="Screenshot 2024-05-10 at 9 18 19 AM" src="https://github.com/Automattic/wp-calypso/assets/28742426/335920a0-288e-423a-8cc5-d4b7a00b9904">


## Testing Instructions



* Run this calypso branch
* test the bulk domains page next at /domains/manage
* verify the checkbox input is reduced to 16px and name font size to 14px.

## Pre-merge Checklist
